kafka_consumer/check.py

```diff
@@ -19,14 +24,11 @@ class KafkaCheck(AgentCheck):
     """
     Check Consumer Lag for Kafka consumers that store their offsets in Zookeeper.
 
-    Modern Kafka consumers store their offset in Kafka rather than Zookeeper,
-    but support for this has not been added yet. It's tricky because this check
-    is much simpler if it assumes a single place can be queried for all consumer
-    offsets, but currently there's no easy way to do that. Once KIP-88 is
-    implemented, it will be much easier to add this functionality, although it
-    would only work for Kafka brokers >= 0.10.2.0. In the meantime, you can
-    instrument your individual kafka consumers to submit their offsets to
-    Datadog.
+    WARNING: Modern Kafka consumer store their offsets in Kafka rather than
+    zookeeper. You can monitor those offsets using the < TODO: TO BE NAMED > check.
```

Comment:
We'll need to update this with the name of the new check whenever that gets figured out